CTBASIC.py

```diff
@@ -83,6 +84,8 @@ def parse_print(line):
             s += chr_(v)
         else:
             s += v.strip('"')
+        if newline:
```

Comment:
This change breaks some of the examples:

    ./ct.py <(./CTBASIC.py examples/LOOP10.BAS )
and
```
xterm -t -e "./ct.py --hold 130 <(./CTBASIC.py examples/ANIMTEST001.bas)"
xterm -t -e "./ct.py --hold 130 <(./CTBASIC.py examples/ANIMTEST002.bas)"
xterm -t -e "./ct.py --hold 130 <(./CTBASIC.py examples/ANIMTEST003.bas)"
```

Need to expand the corresponding `CLEAR`s by 10 bits per `PRINT`

Relates to #22

Reply:
Resolved by adding semicolons `;` to suppress newlines to retain the old `PRINT` behaviour in these examples. No need to adjust `CLEAR` amounts.